package.json

```diff
@@ -19,7 +19,10 @@
     "Frontend/user",
     "packages/shared"
   ],
-  "dependencies": {},
+  "dependencies": {
+    "lodash": "^4.17.21",
+    "rxjs": "^7.8.1"
+  },
```

Comment:
Mấy thư viện này sai ở BE thì thêm vào pakage trong BE á, pagkage ở root này chủ yếu để cho cái mono repository thôi á, hoặc các thư viện sài chung